lib/rule-helpers.js

```diff
@@ -0,0 +1,56 @@
+'use babel';
+
+import { get } from 'request-promise';
```

Comment:
Is there a reason a dependency is being brought in? The [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) is available in Atom. You can see an example usage over in [`linter-jshint` here](https://github.com/AtomLinter/linter-jshint/blob/11052a7271ec83acf302809d7341d57dfd00bc78/lib/helpers.js#L175-L196).

Reply:
Didn't use Fetch API before, is so easy, and no additional deps, great.